### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ralph0211**
A test Pull Request to test image attachments
360_F_220894919_dHe2SOXAdnjI4nt8td3ot4ZhmEmE6WRa

### CODE REVIEW

File: AzurePullRequests.Restore/Enums/RestoreType.cs [hunk 1]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace AzurePullRequests.Restore.Enums
+﻿namespace AzurePullRequests.Restore.Enums
```
Comment (ralph0211) [inline AzurePullRequests.Restore/Enums/RestoreType.cs:1]: Comment in a changed file with a 
images
image

Comment (ralph0211) [reply AzurePullRequests.Restore/Enums/RestoreType.cs:1]: And make it a thread

Comment (raph-backrightup) [reply AzurePullRequests.Restore/Enums/RestoreType.cs:1]: Another user replies

File: AzurePullRequests.Shared/Contracts/Dtos/Comment.cs [hunk 2]
```diff
@@ -4,10 +4,6 @@ public class Comment
     {
         public int Id { get; set; }
 
-        public IdentityRef Author { get; set; }
```
Comment (raph-backrightup) [inline AzurePullRequests.Shared/Contracts/Dtos/Comment.cs:7]: Another user with another comment

Comment (ralph0211) [reply AzurePullRequests.Shared/Contracts/Dtos/Comment.cs:7]: Original user replies
images

Comment (raph-backrightup) [reply AzurePullRequests.Shared/Contracts/Dtos/Comment.cs:7]: Okay
istockphoto-1398462038-612x612

**Pull request closed without merging**